lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -241,7 +242,7 @@ namespace {
 class ExprParentFinder : public ASTWalker {
   friend class ExprContextAnalyzer;
   Expr *ChildExpr;
-  llvm::function_ref<bool(ParentTy, ParentTy)> Predicate;
+  std::function<bool(ParentTy, ParentTy)> Predicate;
```

Comment:
Why is this change required?

Reply:
Because it crashes without this change. `llvm::function_ref` doesn't really support usage like this: http://llvm.org/doxygen/STLExtras_8h_source.html#l00110
> This is intended for use as the type of a function parameter that is not used after the function in question returns.

I think it worked because it didn't capture any outer values. Now, it references outer values `SM` and `ParsedExpr`.